### PR TITLE
ci(helm): fix helm push error

### PR DESCRIPTION
**Pull request by Rory-Z**

### CODE REVIEW

File: deploy/charts/emqx-ee/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 5.0.0
```
Comment (zmstone) [inline deploy/charts/emqx-ee/Chart.yaml:17]: should align with the latest release tag ? which is 5.0.7

Comment (zmstone) [reply deploy/charts/emqx-ee/Chart.yaml:17]: sorry, next version is 5.0.8

Comment (zmstone) [reply deploy/charts/emqx-ee/Chart.yaml:17]: sorry, didn't notice it's for enterprise.

**Review by zmstone — APPROVED** [review]

File: deploy/charts/emqx-ee/values.yaml [hunk 2]
```diff
@@ -0,0 +1,214 @@
+## Default values for emqx.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+## It is recommended to have odd number of nodes in a cluster, otherwise the emqx cluster cannot be automatically healed in case of net-split.
+replicaCount: 3
+image:
+  repository: emqx/emqx-enterprise
```
Comment (Rory-Z) [inline deploy/charts/emqx-ee/values.yaml:8]: @zmstone For EMQX Enterprise 5.0, the docker repository is `emqx/emqx-ee` or `emqx/emqx-enterprise` ?

**Review by zmstone — APPROVED** [review]

**Pull request merged**